Fix wrong SQL in tokenization

kavitharaju opened the pull request:
fixes #216 

The syntax error given by the database was due to the apostrophe in the romanized word injected in the SQL query. 
As we require to run query using soundex, the ORM query builder's `text()` feature is used to inject this part of the query. The romanization library we use, unidecode, adds apostrophes(`'`) within words which was resulting in the error. 

In this fix we remove apostrophes from unidecode's response.

joelthe1 commented:
I think we should sanitize all input to SQL queries so that we don't have any possibility of SQL injection attacks.

kavitharaju commented:
I have changed the way we add the romanized words to the query text. 

For most of the query parts we used SQLAlchemy methods to build the query. As SQLAlchemy doesnot provide functions for soundex and levenshtein, we need to build those parts via RAW text. 

The words added in those texts are not direct user inputs, but processed ones. We take the sentences given by user, remove punctuation, normalize and split them into words(and romanized in two places) and these words are added to the text part of the query. 
Before it was added via python string formatting but now using parameters to the SQLAlchemy text function.

Also, this query is run as a SQLAlchmey query and not directly executed via their `exec` function. So I hope we have enough projection from injections here. Does it look okay to you, @joelthe1 ?

joelthe1 commented:
LGTM 👍 Could you confirm that there are no other instances where are passing raw strings to SQL?

kavitharaju commented:
@joelthe1 .
Yes, I have already changed four other places `text()` was used in that same function. It is not used anywhere else.
